frontend/packages/ux-editor/src/containers/DesignView/DesignView.module.css

```diff
@@ -13,15 +13,16 @@
   margin: var(--margin-top) var(--margin-horizontal) 0 var(--margin-horizontal);
 }
 
-.button {
+.buttonContainer {
   display: flex;
   justify-content: center;
   padding-top: var(--fds-spacing-5);
   background-color: var(--fds-semantic-surface-neutral-subtle);
+  padding-bottom: 50px;
```

Comment:
Hvorfor `padding-bottom`?

Reply:
Kan sjekke, bare flyttet den fra en klasse til en annen for å unngå to klassenavn. Mistenker at den legger på noe padding i bunn av container elementet. Tar en titt.

Reply:
Den var der i utgangspunktet for å få mer plass under knappen for å legge til sider dersom du har flere sider ennn skjermplass.

![image](https://github.com/Altinn/altinn-studio/assets/46874830/f3647a01-a379-4876-b6d1-e0c68ff0284d)


Synes egentlig ikke det ble så ille uten padding heller? 🤔

Reply:
Skjønner, tenker det er greit å ha litt plass der, så den ikke er klint inntil bunnen. Men kanskje vi kunne brukt en av variablene fra designsystemet?